### ISSUE TITLE
Stop work in Release Mode

### ISSUE THREAD

**Issue by amarildolacerda** [procedurallygenerated/flutter_card_io#5]
I compile every time in debug mode and I can run in my phone very well.
When compile with release mode, I cant start the APP on phone.
If remove dependecy, work again in release mode.

I have this line:
\<uses-permission android:name="android.permission.CAMERA" \>, it is not the case.

in dependencies: 
<pre>
 flutter_card_io:
    git:
      url: https://github.com/procedurallygenerated/flutter_card_io.git
</pre>

What more I need to work  in release mode ?

**Comment by AresVampire** [comment]
I am facing this issue too, I have tried several ways but couldn't solve.

**Comment by wsakka** [comment]
What is the error you get?

**Comment by AresVampire** [comment]
Just it is crashing.

**Comment by JohanSnygg** [comment]
I see the same issue, works like a charm in debug mode and release mode when build directly on to the phone using Android Studio, but with any APK installment, either from Store or by manual install on phone, the APP just crashes at startup.

**Comment by guilhermefilippo** [comment]
Any idea to solve this issue?

**Comment by JohanSnygg** [comment]
Yeah, use https://pub.dev/packages/flutter_paycards instead. The package and the recognition is much better and it works good for both Android and iOS.
